docs/components/DocumentationPage.tsx

```diff
@@ -80,17 +79,7 @@ export default class DocumentationPage extends React.Component<Props, State> {
       if (this.layoutRef.current) {
         window.__sidebarScroll = this.layoutRef.current.getSidebarScrollTop();
       }
-      NProgress.start();
     });
-
-    Router.events.on('routeChangeComplete', () => {
```

Comment:
Now im curious if we actually flooded the router events listeners by never detaching 😂

Reply:
Well, the answer is yes 🤷 On every DocumentationPage render, we added another event listeners, which are never cleared

![image](https://user-images.githubusercontent.com/1203991/151367979-c775a30c-cf82-4e3b-97af-765e041dfeb0.png)